### DISCUSSION TITLE
Recommended Kodi Release Level

### DISCUSSION THREAD

**Discussion by mirtim1** [umbrellaplug/umbrellaplug.github.io#227] [Q&A] [answered]
Will Umbrella run alright on Kodi 20.0? Any recommendations on upgrading or staying on 19.5 regarding Umbrella? Anything to be aware of in conjunction with upgrading to Kodi 20 regarding Umbrella?

**Comment by umbrellaplug** [comment, marked as answer]
Umbrella should run fine for you on Kodi 20. I'm happy to report that I have had no major issues reported for compatibility with Kodi 20. Several of the past few updates have been focused on getting small difference between the versions worked out. The biggest change in Kodi 20 is the way items need to have their properties set. The changes required to the code for Kodi 20 have been made. More changes will be pushed to allow for Kodi 21 compatibility as well in the next couple of days.

**Reply by nolaigsoconnor** [reply]
Thank you for your excellent work

**Comment by mirtim1** [comment]
Thank you. 

I appreciate all your work. Umbrella is a great app. I imagine you spend hours a week on this. Do you take donations? If so please post instructions.

Tim

**Reply by umbrellaplug** [reply]
Thanks! I don't actually take any donations or anything. I appreciate it, but I do not want anything for doing this addon. Keeps me safer to be able to say I never made any money from this and it was just a project I shared.

**Comment by mirtim1** [comment]
Have you disabled discussions on GitHub? Can't seem to find the forums anymore. I hope you aren’t giving up supporting this awesome app.

Tim
